### PR TITLE
Update brieflyUI

### PR DESCRIPTION
**Pull request by NobodyKJ**
Complete Front-end functions

### CODE REVIEW

File: src/components/Pages/Content/OutputForm.js [hunk 1]
```diff
@@ -56,47 +123,49 @@ const OutputForm = () => {
...
+        const updatedGoals = prevGoals.filter((goal) => goal.id !== goalId);
+        return updatedGoals;
+        });
   };
 
   let content = (
     <p style={{ textAlign: 'center' }}></p>
   );
 
+<<<<<<< HEAD
```
Comment (LeonQC) [inline src/components/Pages/Content/OutputForm.js]: Here is still conflict need resolve

File: src/components/Pages/Content/OutputForm.js [hunk 2]
```diff
@@ -56,47 +123,49 @@ const OutputForm = () => {
...
+  if (courseGoals != null) {
+=======
   if (courseGoals.length > 0) {
+>>>>>>> 79862af51ff5dfabbb3606d3849d55e3e886ed31
```
Comment (LeonQC) [inline src/components/Pages/Content/OutputForm.js]: Here is still conflict need resolve

File: src/components/Pages/Content/OutputForm.js [hunk 3]
```diff
@@ -56,47 +123,49 @@ const OutputForm = () => {
...
   }
 
-  if (error) {
-    return (
-      <section> 
-      <p>{error}</p>
-      </section>
-    )
-  }
+<<<<<<< HEAD
```
Comment (LeonQC) [inline src/components/Pages/Content/OutputForm.js]: Here is still conflict need resolve

File: src/components/Pages/Content/OutputForm.js [hunk 4]
```diff
@@ -56,47 +123,49 @@ const OutputForm = () => {
...
-    <section> 
-    <p>Loading...</p>
-    </section>
-    )
-  }
+
+render() 
+=======
   
+>>>>>>> 79862af51ff5dfabbb3606d3849d55e3e886ed31
```
Comment (LeonQC) [inline src/components/Pages/Content/OutputForm.js]: Here is still conflict need resolve

File: src/components/Pages/Content/OutputForm.js [hunk 5]
```diff
@@ -109,4 +178,8 @@ const OutputForm = () => {
   );
 };
 
+<<<<<<< HEAD
```
Comment (LeonQC) [inline src/components/Pages/Content/OutputForm.js]: Here is still conflict need resolve

File: src/components/Pages/Content/OutputForm.js [hunk 6]
```diff
@@ -109,4 +178,8 @@ const OutputForm = () => {
...
+export default OutputForm;
+=======
 export default OutputForm;
+>>>>>>> 79862af51ff5dfabbb3606d3849d55e3e886ed31
```
Comment (LeonQC) [inline src/components/Pages/Content/OutputForm.js]: Here is still conflict need resolve

File: src/components/Pages/Content/OutputForm.js (hunk 1 above)
Comment (NobodyKJ) [reply src/components/Pages/Content/OutputForm.js]: Aready detele conflict

File: src/components/Pages/Content/OutputForm.js (hunk 2 above)
Comment (NobodyKJ) [reply src/components/Pages/Content/OutputForm.js]: Aready detele conflict

File: src/components/Pages/Content/OutputForm.js (hunk 3 above)
Comment (NobodyKJ) [reply src/components/Pages/Content/OutputForm.js]: Aready detele conflict

File: src/components/Pages/Content/OutputForm.js (hunk 4 above)
Comment (NobodyKJ) [reply src/components/Pages/Content/OutputForm.js]: Aready detele conflict

File: src/components/Pages/Content/OutputForm.js (hunk 5 above)
Comment (NobodyKJ) [reply src/components/Pages/Content/OutputForm.js]: Aready detele conflict

File: src/components/Pages/Content/OutputForm.js (hunk 6 above)
Comment (NobodyKJ) [reply src/components/Pages/Content/OutputForm.js]: Aready detele conflict

**Review by LeonQC — APPROVED** [review]

**Pull request merged**